lib/client.js

```diff
@@ -150,7 +150,7 @@
 
     Client.prototype.connect = function (options) {
 
-        options = options || {};
+        options = options ?? {};
```

Comment:
I would be interested to know how using `??` here affects browser support.  It may be worth holding off on this syntax specifically in this file if it has a major effect on browser support.

Reply:
This is its browser support:

![image](https://user-images.githubusercontent.com/583657/179695269-433c9701-2083-43bd-a95d-3eb254f94ae2.png)

Taken from: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator#browser_compatibility.

I believe it requires _fairly_ recent browser versions i.e requires chrome 80 and current version is 103. We also lose IE support. According to wikipedia version 80 was released on 2020-02-04. It's not mentioned in the documentation what were the previous browser requirements so we might want to hold on to this one and make the switch in the next breaking change version.

Reply:
Reverted to the previous client code